src/components/Attachment/AttachmentContainer.tsx

```diff
@@ -85,12 +124,31 @@ export const ImageContainer = <
 ) => {
   const { attachment, Image = DefaultImage } = props;
   const componentType = 'image';
+  const imageElement = useRef<HTMLImageElement>(null);
+  const { imageAttachmentSizeHandler } = useChannelStateContext();
+  const [attachmentConfiguration, setAttachmentConfiguration] = useState<
+    ImageAttachmentConfigration | undefined
+  >(undefined);
+
+  useEffect(() => {
```

Comment:
I am thinking that for adjusting the element styles should be used useLayoutEffect but not sure, whether that caused you some problems.